src/query-builder/QueryBuilder.ts

```diff
@@ -1042,6 +1042,10 @@ export abstract class QueryBuilder<Entity> {
                     .slice(1)
                     .join(", ")})`
             case "any":
+                if (driver.options.type === "cockroachdb") {
+                    return `${condition.parameters[0]}::STRING = ANY(${condition.parameters[1]}::STRING[])`
```

Comment:
why do you cast it to string? what if parameter is not a string?

Reply:
Good question; I implemented this while reading through their documentation that suggested casting it to string: https://www.cockroachlabs.com/docs/stable/array.html#supported-casting-and-conversion

Reply:
@AlexMesser what about this comment?